tests/framework/installer/ceph_installer.go

```diff
@@ -753,13 +753,22 @@ spec:
                       #
                       udevadm trigger || true
                       vgimport -a || true
+                      TMP=$(pvs --noheadings --separator=',' -o pv_name,vg_name)
```

Comment:
Please add a bash comment here to explain why it's necessary to prepopulate a lits of PVs instead of inspecting them later after VG removal.

Reply:
Added.